### ISSUE TITLE
[Due for payment 2026-01-30] Screen Reader: Many Pages: The search suggestions are announced without second line of text

### ISSUE THREAD

**Issue by olenamihno** [Expensify/App#77450]
## Action Performed:
Precondition: User is logged in

1. Using VoiceOver on iOS or TalkBack on Android, open the app
2. Swipe to focus the Account tab and double-tap to activate it
3. Swipe to focus the Profile button and double-tap to activate it
4. Swipe to focus the Status menu item and double-tap to activate it
5. Swipe to focus the Vacation delegate menu item and double-tap to activate it
6. Swipe to focus any of the search result suggestions and note how it is announced

This is a global issue affecting all search screens with results that have two lines of text, e.g,.:
1. Search
2. Preferences: Priority Mode
3. Chat screen: the button at the top
4. Group Chat
5. Profile: Timezone

## Expected Result:
Each search suggestion should be announced with all displayed details, including both the name and the email address (or any second line of text), e.g.:
- 'lol owes R$1.00, [EMAIL ADDRESS]'

## Actual Result:
The search suggestions are announced without a second line of text, only the name is announced.

## Area issue was found in:
Profile > Status

## Failed WCAG checkpoints
WCAG 2.4.6 Headings and Labels
https://www.w3.org/WAI/WCAG22/Understanding/headings-and-labels

## User impact:
Descriptive headings are necessary for users who have disabilities that make reading slow and for people with limited short-term memory. These users benefit when section titles make it possible to predict what each section contains. Form input controls with labels that clearly describe the content that is expected to be entered helps users know how to successfully complete the form.

## Suggested resolution:
Ensure the accessibilityLabel of each search suggestion contains both lines of details, e.g.:
- 'lol owes R$1.00, [EMAIL ADDRESS]'
https://appt.org/en/docs/react-native/samples/accessibility-label
https://reactnative.dev/docs/accessibility#accessibilitylabel

## Environment:
Apple iPhone 14 Pro - iOS 18.7.1, Wi-Fi - Native (No mobile browser)
Samsung Galaxy Note10 Plus - Android 12, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos

<img width="1179" height="2428" alt="Image" src="https://github.com/user-attachments/assets/70349489-d83e-4291-b326-34761a792e11" />

https://github.com/user-attachments/assets/ae641b60-d08d-49ed-9673-753404f6a771

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** No
**Logs:** https://stackoverflow.com/c/expensify/questions/4856  NA
**Notes/Photos/Videos:** Any additional supporting documentation  Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7006900
**Issue reported by:** NA
**Slack conversation:** NA
